rotkehlchen/api/rest.py

```diff
@@ -1347,6 +1347,17 @@ def get_assets_mappings(identifier: List[str]) -> Response:
             return api_response(wrap_in_fail_result(str(e)), status_code=HTTPStatus.BAD_REQUEST)
         return api_response(_wrap_in_ok_result(asset_mappings), status_code=HTTPStatus.OK)
 
+    @staticmethod
+    def get_assets_information(identifiers: List[str]) -> Response:
```

Comment:
Is there a need in a new method in rest.py? Maybe just create an `AssetsFilterQuery` at the resource / schema level and call query_all_assets?

Reply:
About creating a new endpoint I did it because the post in the all asset resources for me seemed very oriented to a different logic but after checking it maybe makes sense to reuse it and use it. About putting it in the resources files I believe is better to separate and keep in the in the resources files the logic related to handling the schemas and in the rest put the logic. It helps to keep the structure of the project consistent